### PR TITLE
[TASK-6407] Merging club_lp into develop, "fixed" lots of conflicts

### PR DESCRIPTION
**Pull request by jjramirezn**
I recommend to at least do a smoke test QA before merging this, hopefully there is not too much to fix

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -1,34 +1,44 @@
...
-                        key="chakra-ui-no-flash"
-                        storageKey="chakra-ui-color-mode"
-                    />
-                    <config.PeanutProvider>
-                        <context.ContextProvider>
-                            {children}
+            <body className={`${roboto.variable} ${londrina.variable} ${sniglet.variable} chakra-ui-light font-sans`}>
+                    <PeanutProvider>
+                        <ContextProvider>
+                            <CrispChat />
```
[the remark this answers is not in the document]
Comment (jjramirezn) [reply src/app/layout.tsx]: thanks!

File: package.json [hunk 2]
```diff
@@ -35,10 +37,12 @@
         "eslint-config-next": "^14.2.13",
         "ethers": "5.7.2",
         "framer-motion": "^11.6.0",
+        "frames.js": "^0.15.2",
         "fs-extra": "^11.2.0",
         "i18n-iso-countries": "^7.12.0",
         "install": "^0.13.0",
         "libphonenumber-js": "^1.11.8",
+        "logrocket": "^8.1.0",
```
Comment (Hugo0) [inline package.json:45]: removing

File: package.json [hunk 3]
```diff
@@ -35,10 +37,12 @@
         "eslint-config-next": "^14.2.13",
         "ethers": "5.7.2",
         "framer-motion": "^11.6.0",
+        "frames.js": "^0.15.2",
```
Comment (Hugo0) [inline package.json:40]: also removing

**Review by Hugo0 — APPROVED** [review]
so many changes, jeez

File: src/components/Club/index.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+export * from './faq'
+export * from './features'
+export * from './hero'
+export * from './intro'
+export * from './landing'
+export * from './mike'
+export * from './story'
+export * from './nutsDivider'
+export * from './cashoutSection'
```
[the remark this answers is not in the document]
Comment (jjramirezn) [reply src/components/Club/index.ts:9]: eventually

**Comment by jjramirezn** [comment]
Smoke tested main flows, looks good enough. Merging

**Pull request merged**